### ISSUE TITLE
Main CUL accessibility: Borrowing policies dropdown is missing a label

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#227]
See issue comments here:
https://culibrary.atlassian.net/browse/CULWEBDEVTEAM-1059
